internal/config/validate.go

```diff
@@ -4,13 +4,6 @@ import "fmt"
 
 func Validate(c *Config) error {
 	for _, sql := range c.SQL {
-		sqlGo := sql.Gen.Go
-		if sqlGo == nil {
-			continue
-		}
```

Comment:
The offending `continue`. The Go-specific options validation below moved into codegen.